snap/local/start-exporter.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -eo pipefail # Exit on error
+
+EXPORTER_OPTS=""
+EXPORTER_PATH="/usr/bin/prometheus-postgres-exporter"
+SOCKET_PATH="/tmp"
+
+if [ -z "$SNAP" ]; then
+    # When not running as a snap, expect `DATA_SOURCE_NAME` to be set.
+    if [ -z "$DATA_SOURCE_NAME" ]; then
+        echo "DATA_SOURCE_NAME must be set"
+        exit 1
+    fi
+    exec "$EXPORTER_PATH" $(echo "$EXPORTER_OPTS")
+else
+    # When running as a snap, expect `exporter.user` and `exporter.password`
+    EXPORTER_USER="$(snapctl get exporter.user)"
+    EXPORTER_PASS="$(snapctl get exporter.password)"
+    EXPORTER_PATH="${SNAP}${EXPORTER_PATH}"
+
+    if [[ -z "$EXPORTER_USER" || -z "$EXPORTER_PASS" ]]; then
+        echo "exporter.user and exporter.password must be set"
+        exit 1
+    fi
+
+    DATA_SOURCE_NAME="user=${EXPORTER_USER} password=${EXPORTER_PASS} host=${SOCKET_PATH}"
+    DATA_SOURCE_NAME="${DATA_SOURCE_NAME} port=5432 database=postgres"
```

Comment:
```suggestion
    DATA_SOURCE_NAME+=" port=5432 database=postgres"
```

example:
```
ubuntu@taurus-dev:~$ aaa="test"
ubuntu@taurus-dev:~$ aaa+=" test2"
ubuntu@taurus-dev:~$ echo ${aaa}
test test2
ubuntu@taurus-dev:~$ 
```

Advanced level :-)
```
ubuntu@taurus-dev:~$ declare -a DATA_SOURCE_NAME
ubuntu@taurus-dev:~$ DATA_SOURCE_NAME+=("user=${EXPORTER_USER}")
ubuntu@taurus-dev:~$ DATA_SOURCE_NAME+=("password=${EXPORTER_PASS}")
ubuntu@taurus-dev:~$ DATA_SOURCE_NAME+=("host=${SOCKET_PATH}")
ubuntu@taurus-dev:~$ echo "${DATA_SOURCE_NAME[@]}"
user= password= host=
ubuntu@taurus-dev:~$
```

Reply:
2b613a4